docs/design/design-reviews/recipes/preferred-diagram-tooling.md

```diff
@@ -0,0 +1,7 @@
+# Preferred Diagram Tooling
+
+At each stage in the engagement process, diagrams are a key part of the design review. The preferred tooling for creating and maintaining diagrams is the `.drawio` format from [diagrams.net](http://diagrams.net) (formerly [draw.io](http://draw.io))
```

Comment:
nit: I prefer sentences on newlines in Markdown.

Reply:
oops. done